commit b378af65e75b0d353a83ca7fa5c1ef134e967707 (refs/changes/41/3241/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-01-19T17:06:00-05:00 (1 year, 9 months ago)
    
    HEAD-2385 dhcpd config should use auto.ADMIN_IP instead of dhcpd_admin_ip

diff --git a/sapi_manifests/dhcpd/template b/sapi_manifests/dhcpd/template
index 3cd9c67..bc86c39 100644
--- a/sapi_manifests/dhcpd/template
+++ b/sapi_manifests/dhcpd/template
@@ -29,7 +29,7 @@
   "port": 67,
   "tftpRoot": "/tftpboot",
   "defaultGateway": "{{{compute_node_default_gateway}}}",
-  "serverIp": "{{{dhcpd_admin_ip}}}",
+  "serverIp": "{{{auto.ADMIN_IP}}}",
   "leaseTime": {{{dhcp_lease_time}}},
   "resolvers": {{{resolvers}}}{{^resolvers}}[]{{/resolvers}},
   "netmask": "{{{admin_netmask}}}",
